src/validations/nouns/index.ts

```diff
@@ -0,0 +1,52 @@
+import { getScores } from '../../utils';
+
+export const author = 'waterdrops';
+export const version = '0.1.0';
+
+/**
+ * Nouns Space Validation proposal validation uses:
+ *  
+ * The current validation implementation mutates the "strategies" field of the space
+ * to be able to use proposition power instead of voting power for "nouns-rfp-power".
+ *
+ */
+
+export default async function validate(
+  author: string,
+  space,
+  proposal,
+  options
+): Promise<boolean> {
+  const onlyMembers = options.onlyMembers || space.filters?.onlyMembers;
+  const minScore = options.minScore || space.filters?.minScore;
+  const members = (space.members || []).map((address) => address.toLowerCase());
+  const strategies = [...space.strategies];
+
+  const nounsRFPStrategyIndex = strategies.findIndex(
+    ({ name }) => name === 'nouns-rfp-power'
+  );
+
+  // Use the proposition power instead of the voting power
+  if (nounsRFPStrategyIndex >= 0) {
+    strategies[nounsRFPStrategyIndex].params.powerType = 'proposition';
+  }
+
+  if (members.includes(author.toLowerCase())) return true;
+
+  if (onlyMembers) return false;
+
+  if (minScore) {
+    const scores = await getScores(
```

Comment:
Hey, @curelycue This validation is almost equal to basic validation, right?

except 
```
 if (nounsRFPStrategyIndex >= 0) {
    strategies[nounsRFPStrategyIndex].params.powerType = 'proposition';
  }
```
but for basic validation you can pass your own strategies irrespective of space strategies